ui/raidboss/data/06-ew/raid/p9s.ts

```diff
@@ -465,15 +467,17 @@ const triggerSet: TriggerSet<Data> = {
       netRegex: { id: '817D', source: 'Kokytos', capture: false },
       condition: (data) => data.limitCutDash === 0,
       alertText: (data, _matches, output) => {
+        delete data.lc1DashPlayer;
+        delete data.lc1SoakPlayer;
         const activePlayers = limitCutPlayerActive[data.limitCutDash];
         if (activePlayers === undefined)
           return;
-        const [dashPlayer, soakPlayer] = activePlayers;
-        if (dashPlayer === undefined || soakPlayer === undefined)
+        [data.lc1DashPlayer, data.lc1SoakPlayer] = activePlayers;
+        if (data.lc1DashPlayer === undefined || data.lc1SoakPlayer === undefined)
```

Comment:
I don't understand what the changes to this trigger are doing.  It seems like there's no change here, and the values of dash/soak player are rewritten in other triggers.  Should the changes to this trigger be un-done?

Reply:
Given the changes to Later Dash/Tower Combo (see comment below) and using the vars on `data{}`, I just updated this for consistency.  But using the suggested code below and eliminating the need for the vars on data makes this obsolete, so I'll remove it.